pkg/kubelet/kubelet.go

```diff
@@ -2673,6 +2673,12 @@ func getPodReadyCondition(spec *api.PodSpec, containerStatuses []api.ContainerSt
 			unknownContainers = append(unknownContainers, container.Name)
 		}
 	}
+
+	// In case of unexist unknowContainers, If pod has derminated successed and it has unreadyContainers, just return PodCompleted
+	if podPhase == api.PodSucceeded && len(unknownContainers) == 0 && len(unreadyContainers) != 0 {
```

Comment:
I think we need to ignore unready containers in the case when the pod has succeeded.